### DISCUSSION TITLE
Corvus monedula : 2 birds for the same species ?

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#705] [General]
Hi !

In the French version of BirdNET-Pi, "Corvus Monedula" seems to refer both to the "European Jackdaw" (written in English, even in the French version) and also to "Choucas des Tours" It appears to be 2 different species for the app' but, if I get it right, "Choucas des Tours" and European Jackdaw are the same.

My knowledge of birds is quite weak (I'm just a noob here !) but I think there may be a small mistake here.

Thanks for your help !

**Comment by ehpersonal38** [comment]
Thanks for the report! Should be fixed now: https://github.com/mcguirepr89/BirdNET-Pi/commit/67854086cf75c98704a44e1de6eaa9f4108b5074

**Reply by Fish31520** [reply]
@ehpersonal38 Thanks a lot !
